frontend-instructor/src/components/addCourse/addCourse.jsx

```
@@ -0,0 +1,347 @@
+      
+      if (result.success) {
+        alert("Course added successfully!");
+        navigate("/my-courses");
+      } else {
+        setStatus({ error: `Failed: ${result.message || "Unknown error"}` });
+        alert(`Failed to add course: ${result.message || "Unknown error"}`);
+      }
+    } catch (error) {
+      console.error("Error adding course:", error);
+      setStatus({ error: "Failed to add course" });
+      alert("Failed to add course: " + (error.message || "Unknown error"));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)